3.evaluate_model/get_LOIO_probabilities_all_featurespaces_no_retraining.ipynb

```diff
@@ -0,0 +1,1258 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # define combinations to test over
</pre><p>May also consider storing these in a dictionary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/phenotypic_profiling_model/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Yeah, it might be more elegant to do so. However, it is a relatively minor change that would cause additional compute time that likely isn't worth pursuing. Roshan also uses `itertools.product()` on these combinations, so efficiently iterates over them with limited code, which is one of the primary benefits of dictionary storage. I will skip doing this one, thanks!